lumicks/pylake/tests/test_image.py

```diff
@@ -107,17 +107,18 @@ def size_test(x, axis):
 
 
 @pytest.mark.parametrize(
-    "minimum, maximum, ref_minimum, ref_maximum",
+    "data, minimum, maximum, ref_minimum, ref_maximum",
     [
-        (1.0, 5.0, [1.0, 1.0, 1.0], [5.0, 5.0, 5.0]),
-        ([1.0, 2.0, 3.0], [3.0, 4.0, 5.0], [1.0, 2.0, 3.0], [3.0, 4.0, 5.0]),
-        ([1.0], [3.0, 4.0, 5.0], [1.0, 1.0, 1.0], [3.0, 4.0, 5.0]),
-        ([1.0, 2.0, 3.0], [5.0], [1.0, 2.0, 3.0], [5.0, 5.0, 5.0]),
-        ([1.0, 2.0, 3.0], 5.0, [1.0, 2.0, 3.0], [5.0, 5.0, 5.0]),
+        (np.zeros(10), 1.0, 5.0, [1.0, 1.0, 1.0], [5.0, 5.0, 5.0]),
+        (np.arange(10), 1.0, 5.0, [1.0, 1.0, 1.0], [5.0, 5.0, 5.0]),
+        (np.arange(10), [1.0, 2.0, 3.0], [3.0, 4.0, 5.0], [1.0, 2.0, 3.0], [3.0, 4.0, 5.0]),
+        (np.arange(10), [1.0], [3.0, 4.0, 5.0], [1.0, 1.0, 1.0], [3.0, 4.0, 5.0]),
+        (np.arange(10), [1.0, 2.0, 3.0], [5.0], [1.0, 2.0, 3.0], [5.0, 5.0, 5.0]),
+        (np.arange(10), [1.0, 2.0, 3.0], 5.0, [1.0, 2.0, 3.0], [5.0, 5.0, 5.0]),
```

Comment:
No variance in the data column; move out of parametrize matrix?

Reply:
Not sure I follow, the first one is different (`zeros` vs `arange`)?

Reply:
You're right; I missed that.